spec/jobs/reports/quarterly_account_stats_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+require 'csv'
+
+RSpec.describe Reports::QuarterlyAccountStats do
+  subject(:report) { described_class.new }
+
+  describe '#perform' do
+    let(:end_date) { Time.zone.today }
+    let(:ninety_days_ago) { end_date - 90.days }
+
+    it 'saves the report' do
+      expect(subject).to receive(:report_body).
+        with(ninety_days_ago, end_date).
+        and_return('csv text')
+      expect(subject).to receive(:save_report).with(
+        'quarterly-account-stats',
+        'csv text',
+        extension: 'csv',
+      )
+      report.perform(Time.zone.today)
+    end
+  end
+
+  describe '#report_body' do
+    subject(:report_body) { report.report_body(start_date, end_date) }
+
+    let(:start_date) { Time.zone.today - 90.days }
+    let(:end_date) { Time.zone.today - 1.day }
```

Comment:
Looking at this with fresh eyes, we should pull `Time.zone.today` into a single `let`:

```suggestion
    let(:today) { Time.zone.today }
    let(:start_date) { today - 90.days }
    let(:end_date) { today - 1.day }
```
(and update all references to `Time.zone.today`